test/xpu/skip_list_common.py

```diff
@@ -1629,21 +1629,16 @@
         "test_linalg_lstsq_input_checks_xpu_complex64",
         "test_linalg_lstsq_input_checks_xpu_float32",
         "test_linalg_lstsq_input_checks_xpu_float64",
-        "test_norm_fused_type_promotion_xpu_bfloat16",
-        "test_norm_fused_type_promotion_xpu_float16",
         "test_dot_invalid_args_xpu",
         "test_vdot_invalid_args_xpu",
+        # RuntimeError: Fail to enable Kineto Profiler on XPU due to error code: 200
+        "test_norm_fused_type_promotion_xpu_bfloat16",
+        # AssertionError: True is not false
+        "test_norm_fused_type_promotion_xpu_float16",
```

Comment:
Should file an issue for it?

Reply:
Please comment the skip with issue link.

Reply:
done.